game.js

```diff
@@ -0,0 +1,143 @@
+var sizeOfOneSquare = 40, sizeOfFoundation = 3;
+var canvas = document.getElementById("main"), context = 
+canvas.getContext("2d");
+function drawmap() {
+    for (var i=0; i<sizeOfFoundation; i++){
+        for (var j=0; j<sizeOfFoundation; j++){
+            context.strokeRect(0+i*sizeOfOneSquare, 0+j*sizeOfOneSquare, 
```

Comment:
what is 0 + number? what the purpose of 0?

Reply:
show the begin of page

Reply:
maybe you can del it?